deploy/k8s/manifest.yaml

```diff
@@ -363,15 +375,57 @@ data:
           - include: k8s.container_spec_cpu_quota
             action: insert
             match_type: regexp
-            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's Memory usage
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's CPU quota
             experimental_match_labels: { "image": "^$", "pod": "(.|\\s)*\\S(.|\\s)*", "namespace": "(.|\\s)*\\S(.|\\s)*" }
             new_name: k8s.pod.spec.cpu.quota
           - include: k8s.container_spec_cpu_period
             action: insert
             match_type: regexp
-            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's Memory usage
+            # empty `image` label and non-empty `pod` and `namespace` are datapoints of Pod's CPU period
             experimental_match_labels: { "image": "^$", "pod": "(.|\\s)*\\S(.|\\s)*", "namespace": "(.|\\s)*\\S(.|\\s)*" }
             new_name: k8s.pod.spec.cpu.period
+          - include: k8s.container_network_receive_bytes_total
```

Comment:
the same for my second point in other comment apply here (however grouping is correct here)